### DISCUSSION TITLE
code-server-3.8.0-linux-amd64#

### DISCUSSION THREAD

**Discussion by ivanjeremic** [coder/code-server#2510] [General]
I try to use `code-server-3.8.0-linux-amd64#` on `Ubuntu LAMP on Ubuntu 20 LAMP server` everything is fine and runs in the console but in the Browser all I get is 
```
This site can’t be reached46.101.145.166 refused to connect.
```

Any ideas why?

**Comment by code-asher** [comment]
What does your Apache config look like and how is code-server being
invoked?

**Reply by ivanjeremic** [reply]
Good question, I'm not an Apache or Linux guy but I try to learn along the way, I used the standard LAMP install from DigitalOcean Marketplace for 5$/m

**Reply by code-asher** [reply]
I think you'll need to edit the Apache configuration so it'll proxy to code-server.

Example config: https://github.com/cdr/code-server/issues/1544#issuecomment-743912980

How to add/edit the configuration: https://marketplace.digitalocean.com/apps/lamp
